src/azure-cli/azure/cli/command_modules/vm/_vm_utils.py

```diff
@@ -141,6 +144,7 @@ def normalize_disk_info(image_data_disks=None,
     data_disk_sizes_gb = data_disk_sizes_gb or []
     image_data_disks = image_data_disks or []
 
+    data_disk_delete_option = validate_delete_options(attach_data_disks, data_disk_delete_option)
```

Comment:
When `data_disk_delete_option` is a string type, calling `validate_delete_options` method will make `data_disk_delete_option` become None. May I ask is this in line with the expectation?

```
def validate_delete_options(resources, delete_option):
    """ Extracts multiple space-separated delete_option in key[=value] format """
    if resources and isinstance(delete_option, list):
        ......
        return delete_option_dict
    return None
```

Reply:
it would never be string type as I declared `nargs='+'`, which will make sure it is list type.

Reply:
OK, got it~